packages/office-ui-fabric-react/src/components/DetailsList/DetailsColumn.styles.ts

```diff
@@ -88,6 +88,9 @@ export const getStyles = (props: IDetailsColumnStyleProps): IDetailsColumnStyles
             display: 'block'
           }
         }
+      },
+      {
+        border: 'undefined'
```

Comment:
I _think_ this can be `undefined` and it will accomplish the same thing. Our CSS lib must unstringify these strings hence why it is showing up as `undefined` in the snapshots below.

I would consider using the `undefined` keyword and make sure the output is the same. If so, then use that.

Reply:
The reason I suggest this is because _I don't think_ that `'undefined'` is a valid CSS value.

![image](https://user-images.githubusercontent.com/706967/49042661-96eb2900-f17d-11e8-8878-563216fadc4a.png)

Reply:
I'm a bit hesitant to merge this without understanding better. 

`'undefined'` isn't a valid CSS value right? If that's the case, then this works because it circumvents the Object merging `merge-styles` is doing, but ends up omitting the property due to the value being `undefined`?

Are we outputting `border: 'undefined'` in the final CSS for this selector?

Reply:
I _do_ appreciate the specificity issue that this PR is addressing. But I want to make sure what is happening here is clear for folks reading the code.

Reply:
> Are we outputting border: 'undefined' in the final CSS for this selector?

No, this does not happen. Probable due to the reason you mentioned above. But as to why `undefined` and `'undefined'` are behaving differently is puzzling. 
> Let me talk with folks that understand merge-styles better than I in the AM. cc: @dzearing.

Yes, makes sense. Let's wait for David to get back on this.

Reply:
@dzearing  @kenotron  Do you guys have any opinion on this? `border : 'undefined'` turned out to be a case of an invalid property which essentially doesn't get converted to any valid css property , thus solving the override issue. 

Since this isn't the right way, following are the other ways I could think of which would solve the problem:
- Separate out `border:none` from `cellStyles` in detailsHeader.styles and pass it separately in the places it is being used. 

- Use `!important` with the  properties in `borderWhileDragging` 

- Use a less specific selector without the parent class name, i.e `&.ms-DetailsHeader-cell`

Reply:
@KevinTCoughlin, I tried approach #1 from above. This looks like a much better fix, if you feel it's not, then kindly suggest one of either of #2 or #3, or maybe some other better way?
Thanks.